### ISSUE TITLE
django rest not cache

### ISSUE THREAD

**Issue by fangaofeng** [noripyt/django-cachalot#147]
## What happened?
when i use cachalot , admin url visit can see cache in redis.But when drf api visit, not see cache.

## What should've happened instead?
When rest api use, can see data in redis.

[//]: # (Include host configuration: OS, Django version, database, etc.)
django 2.2
drf 3.10
cachalot is last new.
settings
```
CACHES = {
    'default': {
        'BACKEND': 'django_redis.cache.RedisCache',
        "LOCATION": "redis://127.0.0.1:6379/1",  # env('REDIS_URL'),
        'OPTIONS': {
            'CLIENT_CLASS': 'django_redis.client.DefaultClient',
            'CONNECTION_POOL_KWARGS': {"max_connections": 100, "retry_on_timeout": True},
            # Mimicing memcache behavior.
            # http://niwinz.github.io/django-redis/latest/#_memcached_exceptions_behavior
            'IGNORE_EXCEPTIONS': True,
        }
    }
}
CACHALOT_ENABLED = env.bool('CACHALOT_ENABLED', default=False)
CACHALOT_TIMEOUT = env('CACHALOT_TIMEOUT', default=60 * 15)
# CACHALOT_DATABASES = ['default']
CACHALOT_ONLY_CACHABLE_TABLES = [
    'admin_interface_theme', 'course_coursetype', 'permissions_operation', 'permissions_role',
    'permissions_rolefilterbackendmodel', 'permissions_roleoperationshipwithfilter',
    'permissions_roleoperationshipwithfilter_filters', 'orgs_department', 'traingroup_traingroup',
    'traingroup_traingroup_trainers', 'authtoken_token']
CACHE_MIDDLEWARE_SECONDS = 600 * 15

```

**Comment by Andrew-Chen-Wang** [comment]
@fangaofeng I'm slightly confused. Can you clarify what you're not seeing? I'm not aware of a debug toolbar that the DRF team created. Your settings look OK, and it doesn't seem like you're using django-debug-toolbar. Can you please install Django debug toolbar, follow the documentation, and report back if there is performance issues when you send a request via the website vs. via the API?

TL;DR Install Django-debug-toolbar, add cachalot panels, and report back with some performance results. 

Cachalot essentially hacks your querysets and caches your queries automatically. To have it not work here doesn't make sense in my opinion, so some performance results would be really helpful!

**Comment by beda42** [comment]
Hello, just a quick note - I am seeing cache not being used (in DRF views, but that is probably not relevant) when I use the CACHALOT_ONLY_CACHABLE_TABLES setting. Even though all tables from a query are included, the result is not cached. If I disable CACHALOT_ONLY_CACHABLE_TABLES, evetything works ok.
I just found this issue and will investigate further, but maybe this might point to the right direction with this issue.

**Comment by beda42** [comment]
After some digging, I found out where the problem was for me. In `_get_tables`, the set of tables that are checked for "cachability" is seeded from `query.table_map`. However, for some reason, in my case this map contains 6 tables even though only 2 actually appear in the query itself. This is probably related to foreign keys.

I think that @fangaofeng should try to include more related tables into CACHALOT_ONLY_CACHABLE_TABLES to see if it helps.

Also, I am not sure if this is something that I should report as separate bug.

**Comment by Andrew-Chen-Wang** [comment]
I think it’d be nice to have a command that checks for these kinds of mistakes by checking cache only and don’t cache frozensets. Thought? Willing to help out? @beda42

**Comment by fangaofeng** [comment]
sorry，answer so later.i check debug tools bar,there is nothing .But when admin visited, show a lot of trace for query.

**Comment by Andrew-Chen-Wang** [comment]
Please update to the latest version of django-cachalot

**Comment by fangaofeng** [comment]
ok，let me try

**Comment by Andrew-Chen-Wang** [comment]
@fangaofeng Any updates?

**Comment by Andrew-Chen-Wang** [comment]
Stale with no response
